draft-ietf-httpbis-http2.xml

```diff
@@ -732,7 +732,7 @@ HTTP2-Settings    = token68
           <x:ref>CONTINUATION</x:ref> frames MUST reassemble header blocks and perform decompression
           even if the frames are to be discarded.  A receiver MUST terminate the connection with a
           <xref target="ConnectionErrorHandler">connection error</xref> of type
-          <x:ref>COMPRESSION_ERROR</x:ref> if it does not decompress a header block.
+          <x:ref>COMPRESSION_ERROR</x:ref> if it encounters a decoding error or does not decompress a header block.

```

Comment:
I understand the sentiment, but I'd prefer a separate statement about decoding errors.  Including this text here might confuse things unnecessarily.